lib/PreconditionRunner.pm

```diff
@@ -95,12 +96,53 @@ sub write_num_passes
 sub run_to_steady_state($)
 {
     my $self = shift;
-    
     my %args = @_;
 
     my $msg_prefix = $args{'msg_prefix'};
-    my $output_dir = $args{'output_dir'};
     my $test_ref = $args{'test_ref'};
+    my $output_dir = $args{'output_dir'};
+
+    # preconditioner.exe does not support multiple files
```

Comment:
It's a new area, so I haven't seen any data for the effect.  But, I bet you can think of it like several independent drives.  Say, two for an example, and you're driving a random write workload to each.  Each needs to get fragmented before we enter the test phase.  Stream A might fragment faster (maybe because it's 8kB writes and stream B is 4kB writes).  If we used the preconditioner, it would exit and the drive might start cleaning up stream A's blocks while stream B is still preconditioning.